src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,205 @@
+"use client";
+
+import React from "react";
+import { Svg, Document, Page, Text, View, StyleSheet, Path } from "@react-pdf/renderer";
+import { faFlag, faSquare, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+import { ParcelOfSpecificDateAndLocation } from "@/components/DayOverview/DayOverview";
+
+interface DayOverviewDateProps {
+    date: Date;
+}
+
+interface DayOverviewLocationProps {
+    location: string;
```

Comment:
Do all of these need to be interfaces? Instead of just having it written out once in the props?

Reply:
I think Tom likes this format more